pkg/validation/number.go

```
@@ -0,0 +1,8 @@
+package validation
+
+import "regexp"
+
+func OnlyDigits(number string) bool {
+	ok, _ := regexp.MatchString("\\d+", number)
```

На бою недопустимо игнорирование ошибок

---

Исправил через MustCompile() - пойдем по пути fail fast